### ISSUE TITLE
Segment issue in API server

### ISSUE THREAD

**Issue by aabedraba** [kubeshop/kusk-gateway#609]
Found these logs while looking at the API server for Kusk: 

```
$ kubectl logs -n kusk-system deployments/kusk-gateway-api -f
segment 2022/08/15 12:31:07 ERROR: sending request - Post "https://api.segment.io/v1/batch": x509: certificate signed by unknown authority
segment 2022/08/15 12:31:07 ERROR: 1 messages dropped because they failed to be sent and the client was closed
```

**Comment by jasmingacic** [comment]
How can I replicate this?

**Comment by aabedraba** [comment]
**Replicate**: 

1- Install Kusk on GKE with analytics enabled
2- Read logs from `kusk-gateway-envoy-fleet`

`kubectl logs -n kusk-system deployments/kusk-gateway-api -f`

3- Run an operation through the API like 

```sh
cat << EOF | curl -X POST -d @- **EXTERNAL_IP**
{
  "name": "new-api",
  "namespace": "default",
  "envoyFleetName": "kusk-gateway-envoy-fleet",
  "envoyFleetNamespace": "kusk-system",
  "openapi": "openapi: 3.0.0"
}
EOF
```

4- In the logs, the original error should appear

**Comment by aabedraba** [comment]
FYI: 

We are not disabling analytics on installation, which should not be the case: https://github.com/kubeshop/kusk-gateway/blob/main/cmd/kusk/cmd/install.go#L346-L359

We should disable it just like we do for Kusk Gateway: https://github.com/kubeshop/kusk-gateway/blob/main/cmd/kusk/cmd/install.go#L299

So `ANALYTICS_ENABLED=false` is not working with the API server as far as I know. @jasmingacic

**Comment by jasmingacic** [comment]
@kylehodgetts can you take a look at helm-charts. It seems we are not passing analytics flag to Dashboard API

**Comment by jasmingacic** [comment]
Fixed with #608
